airflow/providers/google/cloud/operators/pubsub.py

```diff
@@ -602,7 +602,8 @@ class PubSubPublishMessageOperator(GoogleCloudBaseOperator):
               'attributes': {'type': 'greeting'}
              }
         m2 = {'data': b'Knock, knock'}
-        m3 = {'attributes': {'foo': ''}}
+        m3 = {'data': 'Who's there?'}
```

Comment:
Added string-type message data example to PubSubPublishMessageOperator docstring.